admin/cluster_setup.py

```diff
@@ -193,14 +217,25 @@ def main(reactor, args, base_path, top_level):
                 print("Didn't finish creating the cluster.")
                 runner.stop_cluster(reactor)
             else:
-                print("The following variables describe the cluster:")
                 environment_variables = get_trial_environment(cluster)
+                environment_strings = list()
                 for environment_variable in environment_variables:
-                    print("export {name}={value};".format(
-                        name=environment_variable,
-                        value=shell_quote(
-                            environment_variables[environment_variable]),
-                    ))
+                    environment_strings.append(
+                        "export {name}={value};\n".format(
+                            name=environment_variable,
+                            value=shell_quote(
+                                environment_variables[environment_variable]
+                            ),
+                        )
+                    )
+                environment = ''.join(environment_strings)
+                print("The following variables describe the cluster:")
+                print(environment)
```

Comment:
The code before my change did exactly that. With my change the variables are not only printed, but are also saved  to a file (see the next 5 lines).  So, I thought that it made sense to build the full content and then to print it and to save it rather than printing it and appending it to a file piecemeal.